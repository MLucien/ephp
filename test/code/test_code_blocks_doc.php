<?php if ( ! $user_table ): ?>
    <tr class="form-field form-required user-pass1-wrap">
    <th scope="row">
<?php endif;

