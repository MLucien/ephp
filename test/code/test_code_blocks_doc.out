
Notice: Undefined variable: user_table in {{CWD}}/test/code/test_code_blocks_doc.php on line 1
    <tr class="form-field form-required user-pass1-wrap">
    <th scope="row">
